torch/_dynamo/variables/builder.py

```diff
@@ -176,32 +176,16 @@ def __call__(self, value):
     @staticmethod
     @functools.lru_cache(None)
     def _common_constants():
-        return set(range(17)).union(
```

Comment:
wtf was this doing

Reply:
Specialize every number that is less than 17 lol. This is what I mean by "int unspecialization didn't work"